pkg/cloud/cloud.go

```diff
@@ -27,3 +31,13 @@ type CloudVMInstance struct {
 }
 
 type InstanceIdentifier string
+
+// ValidateTaskRunId ensures that a TaskRunId string is in the following format: '<TaskRun Namespace>:<TaskRun Name>'
+func ValidateTaskRunId(taskRunId string) error {
+	idComponents := strings.Split(taskRunId, ":")
+	if len(idComponents) == 2 {
+		return nil
+	}
```

Comment:
As a lighter alternative, instead of splitting you can count the occurrences with `strings.Count(taskRunId, ":")`

```suggestion
	if strings.Count(taskRunId, ":") == 1 {
		return nil
	}
```

Reply:
Implemented!